### DISCUSSION TITLE
No songs loading/playing after 3.8.0 update (macOS)

### DISCUSSION THREAD

**Discussion by Mateus109** [KRTirtho/spotube#1780] [Q&A]
Since the 3.8.0 update, Spotube simply doesn't work anymore.  I can open the app, log in to Spotify, but Spotube will not load/play any tracks.  I've removed the app and cleared all the associated files/folders, but still the same problem. It worked perfectly prior to the 3.8.0 update.

UPDATE:  It randomly started working, I played 2 songs, and now it's stopped again.  Strange!

**Comment by KRTirtho** [comment]
What's your macOS version?

**Reply by Mateus109** [reply]
macOS 14.6.1

I've not had the problem, so far, today. Maybe it was not an issue to the Spotube app but YouTube/Spotify temporarily blocking something?

**Reply by KRTirtho** [reply]
Ok, open Finder, press `cmd + shift + .` to show hidden folders.
Then, go to Home > Library > Containers. Then Delete the `spotube` or `oss.krtirtho.spotube` folder

Also delete, `Home > Library > Application Support > oss.krtirtho.spotube` folder

Or, run following in the terminal:

```shell
cd ~/Library/Containers && rm -rf ./oss.krtirtho.spotube && cd ~/Library/Application\ Support/ && rm -rf ./oss.krtirtho.spotube
```

**Comment by Filippide8864** [comment]
I also have the same problem but nothing is solved, macOS Sonoma 14.5.
